app/views/admin/basic_projects/_form.html.slim

```diff
@@ -14,19 +14,21 @@
     .view-element = @project.try(:name)
     = f.input_field :name
 
+  = f.input :division
+    .view-element = @project.division.try(:name)
```

Comment:
Good use of `try` here. 

I'm not sure why it's used on the existing code for `:name`, `:status`, and `:signing_date` though. We can assume `@project` won't be nil. Could you please refactor those.